src/generic.rs

```diff
@@ -67,7 +71,8 @@ where
     T: Beef + ?Sized,
     U: Capacity,
 {
-    const fn borrowed_from_parts(ptr: NonNull<T::PointerT>, fat: usize, cap: U::Field) -> Self {
+    // fixme: it should be unsafe :)
+    const fn from_parts(ptr: NonNull<T::PointerT>, fat: usize, cap: U::Field) -> Self {
```

Comment:
@maciejhirsz uses a lot of unsafe in the code, believing that `Cow` is created only in `borrowed`/`owned`
In this case, it is reasonable to make `from_parts` `unsafe`